src/provider/rpc.ts

```diff
@@ -242,7 +242,7 @@ export class RpcProvider implements ProviderInterface {
     return this.fetchEndpoint('starknet_estimateFee', {
       request: {
         type: 'INVOKE',
-        contract_address: invocation.contractAddress,
+        sender_address: invocation.contractAddress,
```

Comment:
Note that the entry parameter is contractAddress but used as sender_address.
This is left to be like this because of Sequencer.